content/en/changelogs/1.30.0-changelog.md

```diff
@@ -61,26 +61,26 @@ expression:
 
 If you have a pipeline with multiple triggers using different artifact constraints/expected artifacts, these have for a while been evaluated in an unexpected matter.
 
-![Artifact constraints](artifact_constraints.png)
+![Artifact constraints](assets/1.30.0/artifact_constraints.png)
```

Comment:
Thanks so much for doing this!  We didn't check carefully enough when assembling the release notes.

The links are unfortunately still broken...but close!

On master, the current link to this image is https://spinnaker.io/changelogs/1.30.0-changelog/artifact_constraints.png.  The new link with this is https://deploy-preview-321--spinnaker-io.netlify.app/changelogs/1.30.0-changelog/assets/1.30.0/artifact_constraints.png.  I'm not quite sure the the best way to unwind this...it feels a little funny to have a `../` in the path, but that could be the way.  Unless you beat me to it, I'm going to push a commit that tries this out.